package.json

```diff
@@ -28,7 +28,16 @@
     "extends": [
       "react-app",
       "react-app/jest"
-    ]
+    ],
+    "rules": {
+      "quotes": [
+        "error",
+        "single",
```

Comment:
Ah the irony

Reply:
One day we'll be able to write JSON with single quotes